main.py

```diff
@@ -11,7 +11,11 @@
 TOKEN = os.getenv("BOT_TOKEN")
 PREFIX = constants.COMMAND_PREFIX
 
-client = commands.Bot(commands.when_mentioned_or(PREFIX), case_insensitivity=True, owner_id=688275913535914014)
+SUPPORT_SERVER = "https://discord.gg/CgH6Sj6"
+INVITE = "https://discord.com/api/oauth2/authorize?client_id=715545167649570977&permissions=980675863&scope=bot"
+
+client = commands.Bot(commands.when_mentioned_or(
+    PREFIX), case_insensitivity=True, owner_id=688275913535914014)
```

Comment:
```suggestion
client = commands.Bot(commands.when_mentioned_or(PREFIX), case_insensitivity=True, owner_id=688275913535914014)
```
Our line limit is 150, which this line is well within, there is no reason to split it.

Reply:
I literally have no idea why that has happened.